WordPress/gutenberg
# [RNMobile] Allow shrink toolbar

## Description

Fixes: https://github.com/wordpress-mobile/gutenberg-mobile/issues/2180

Ref to:
gutenberg-mobile: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2220

That PR introduces the possibility to stack options in <BlockMobileToolbar /> into one toolbar button which will open an ActionSheet (ios) / BottomSheet (Android).

## How has this been tested?

1. Open mobile app
2. Add Buttons block
3. Focus Button block inside Buttons block
4. Expect to see stacked options in the mobile toolbar
5. Press button in the mobile toolbar
6. Expect the disabled button `Move Left` for the first button
7. Expect the disabled button `Move Right` for the last button
8. Try all the options:
	* expect Button is moving left/right when these options are available
	* expect to be able to open settings
	* expect to be able to remove block

## Screenshots 

ios | android
--- | ---
<img width="409" alt="Screenshot 2020-05-05 at 16 57 15" src="https://user-images.githubusercontent.com/22746080/81157890-bb108380-8f87-11ea-9028-9ec91a74504a.png"> | <img width="427" alt="Screenshot 2020-05-05 at 14 09 44" src="https://user-images.githubusercontent.com/22746080/81157905-be0b7400-8f87-11ea-8a87-68aa8a92df8d.png">

## Types of changes

* The new type of mobile toolbar named `ShrinkBlockMobileToolbar` along with new prop `allowShrinkToolbar`.

* Add option to disable buttons within Android Picker.

* Two new options to iOS Picker called: `disabledButtonIndices` and `destructiveButtonIndex`


## Checklist:
